### ISSUE TITLE
VoteParty 

### ISSUE THREAD

**Issue by xXIDontKnowXx** [esquerbatua/DestinyumCraft#24]
Ya sabes en lo que consiste

**Comment by esquerbatua** [comment]
Sí, el problema es que en cada reinicio del server el plugin que registra los votos se resetea por así decirlo y no guarda el total de votos, aun así podría funcionar.
